templates/settings.py.erb

```diff
@@ -23,6 +23,10 @@ REDIS_URL = "redis://localhost:<%= scope['redis::port'] %>/<%= scope['pulpcore::
 
 USE_NEW_WORKER_TYPE = <%= scope['pulpcore::use_rq_tasking_system'] ? "False" : "True" %>
 
+<% if scope['pulpcore::worker_ttl'] -%>
+WORKER_TTL = <%= scope['pulpcore::worker_ttl'] %>
+<% end -%>
+
```

Comment:
Thanks. Looking at it a bit more, arguably the newline above should be removed so that `USE_NEW_WORKER_TYPE` and `WORKER_TTL` exist in the same block. I'll take that approach.